extensions/notebook/src/extension.ts

```diff
@@ -7,22 +7,31 @@
 
 import * as vscode from 'vscode';
 import * as sqlops from 'sqlops';
+import * as os from 'os';
+import * as fs from 'fs-extra';
 import * as nls from 'vscode-nls';
 const localize = nls.loadMessageBundle();
 
 let counter = 0;
+const notebookConfigKey = 'notebook';
+const sqlKernelEnablebConfig = 'sqlKernelEnabled';
+const pythonPathConfigKey = 'pythonPath';
+const SQL_NOTEBOOK_PROVIDER = 'sql';
+const DEFAULT_NOTEBOOK_PROVIDER = 'builtin';
+const JUPYTER_NOTEBOOK_PROVIDER = 'jupyter';
```

Comment:
We may move these to a Constants file since they are likely to be re-used by other classes as well?